README.md

```diff
@@ -154,6 +155,75 @@ export default function () {
 }
 ```
 
+## Helpers
+
+The `xk6-kubernetes` extension offers helpers to facilitate common tasks when setting up a tests. All helper functions work in a implicit namespace to facilitate the development of tests segregated by namespace. The helpers are accessed using the following methods:
+
+|  Method      | Parameters|   Description |
+| -------------| ---| ------ |
+| helpers      | | returns helpers that operate in the default namespace |
+| namespacedHelpers | namespace | returns helpers that operate in the given namespace |
```

Comment:
Having both the `helpers` _and_ `namespacedHelpers` feels awkward. I'd prefer `default` namespace be used unless explicitly provided to `helpers()`.